azurion/properties/Requirement7.mcf

```diff
@@ -0,0 +1,3 @@
+exists a:Arm.exists dirJ:JoystickDirection.[true*.moveArmJoystick(a, dirJ)]mu X. ([!moveArm(a, translateJoystickToArmMovement(dirJ))]X && <true>true)
```

Comment:
This should happen only if ``emergencyMode`` is ``false``.
Same remark with the ``forall`` instead of ``exists``.